h2o-genmodel/src/main/java/hex/genmodel/CommonRowtoRawDataConverterProvider.java

```diff
@@ -0,0 +1,24 @@
+package hex.genmodel;
+
+import hex.genmodel.easy.CategoricalEncoder;
+import hex.genmodel.easy.EasyPredictModelWrapper;
+import hex.genmodel.easy.RowToRawDataConverter;
+
+import java.util.Map;
+
+public class CommonRowtoRawDataConverterProvider implements RowtoRawDataConverterProvider {
```

Comment:
I would prefer DefaultRowtoRawDataConverterProvider over Common*
other than that I think this is great